app/shared/date-picker/_date-picker.scss

```diff
@@ -17,17 +17,21 @@
 
   input {
     width: 100%;
-    border: 1px solid $input-border;
-    padding: 5px;
+
+    &:focus {
+      outline: none
+    }
```

Comment:
For accessibility's sake, it's better to keep the outline on. 
Even if it doesn't look pretty, we can play around with its color.

Reply:
Really ? ._. ok then